test/storage.test.js

```diff
@@ -0,0 +1,36 @@
+import {
+  projectList,
+  storeTodos,
+  projectFactory,
+  createProjects,
+  removeProject,
+} from '../src/storage';
+
+describe('Todo display tests', () => {
+  storeTodos();
+  projectFactory();
+  createProjects();
+  removeProject();
+
+  test('Display default todo', () => {
+    const defaultProject = projectList.push('Default Project');
+    for (let i = 0; i < defaultProject.length; i += 1) {
+      if (defaultProject[i] === 'Default project') {
+        expect(defaultProject[i].toBe('Default project'));
+      }
+    }
+  });
+  test('Creates a new project, createProjects()', () => {
```

Comment:
- You should be calling `addProjectToList` method to add the new project to the `projectList` array. Then test whether the length of `projectList` is greater. Checking for title `Morning Mommy` seems wrong because it's not in `projectObjectListTitles`